doc/source/analyzing/filtering.rst

```diff
@@ -86,7 +86,34 @@ other data object to generate images, examine its values, etc.
 
 .. notebook:: mesh_filter.ipynb
 
-Cut regions can also operator on particle fields, but a single cut region object
+In addition to inputting string parameters into cut_region to specify filters,
+wrapper functions exist that allow the user to use a simplified syntax for
+filtering out unwanted regions. Such wrapper functions are methods of
+:func: ``YTSelectionContainer3D``.
+
+.. notebook-cell::
+
+   import yt
+   ds = yt.load('Enzo_64/DD0042/data0042')
+   ad = ds.all_data()
+   overpressure_and_fast = ds.include_above('pressure', 1e-14)
+   # You can chain include_xx and exclude_xx to produce the intersection of cut regions
+   overpressure_and_fast = overpressure_and_fast.include_above('velocity_magnitude', 1e2, 'km/s')
+
+   print('Density of all data: ad["density"] = \n%s' % ad['density'])
+   print('Density of "overpressure and fast" data: ad["density"][overpressure_and_fast] = \n%s' %
+          ad['density'][overpressure_and_fast])
+
+The following exclude and include functions are supported:
```

Comment:
Might want to also add mentions to `<include/exclude<_<above/below>`.

Reply:
Done and done!